rwatch/ui/layer/simple_menu_layer.c

```diff
@@ -0,0 +1,110 @@
+/* simple_menu_layer.h
+ * routines for the simple menu layer
+ * libRebbleOS
+ *
+ * Author: Barry Carter <barry.carter@gmail.com>
+ * Author: Hermann Noll <hermann.noll@hotmail.com>
+ */
+
+#include "simple_menu_layer.h"
+#include "menu_layer.h"
+
+static uint16_t prv_simple_menu_get_number_of_sections(struct MenuLayer* menu_layer, void* callback_context) {
```

Comment:
I usually call local functions `_get_number_of_sections`, but this is OK too.  Also, `struct MenuLayer *menu_layer`, `void *callback_context`.